addons/hr_recruitment_sms/views/hr_applicant_views.xml

```diff
@@ -1,16 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <odoo>
-    <record id="action_hr_applicant_mass_sms" model="ir.actions.act_window">
+    <record id="action_applicant_send_sms" model="ir.actions.server">
```

Comment:
Keep original ir.actions.act_windows, and in ``action_send_sms`` reuse it like this:
```py
    def action_get_attachment_view(self):
        self.ensure_one()
        res = self.env['ir.actions.act_window']._for_xml_id('base.action_attachment')
        res['domain'] = [('res_model', '=', 'approval.request'), ('res_id', 'in', self.ids)]
        res['context'] = {'default_res_model': 'approval.request', 'default_res_id': self.id}
        return res
```

Reply:
Also, to execute an ``ir.actions.server`` you need to edition access right on the record. Are you sure this is always the case for team leaders ?